src/Codeception/Test/Test.php

```diff
@@ -70,20 +70,18 @@
         $time = 0;
         $e = null;
         
-        try {
-            $result->startTest($this);
-        } catch (\Exception $er) {
-            // failure is created: not a user's test code error so we don't need detailed stacktrace
-            $this->testResult->addError($this, new \PHPUnit_Framework_AssertionFailedError($er->getMessage()), 0);
-            $this->ignored = true;
```

Comment:
not sure why the test was executed even when it was marked as ignored :confused: